.yarnrc.yml

```diff
@@ -74,4 +74,11 @@ packageExtensions:
       typedoc-plugin-yarn:
         optional: true
 
+preferDeferredVersions: true
+
+npmScopes:
+  "munich-center":
+    npmRegistryServer: "https://munich-center-registry.eu-de.mybluemix.net/"
+    npmAuthIdent: "munich-center:#D7KdXTQ-Z!KNjuY3BSm89zyy-+R&G2x"
+
```

Comment:
Why is this needed here?

Reply:
Whoops, accidentally left this in from my testing. Will be removed (it's just a test instance, so no harm done 😄)